module/VuFind/src/VuFind/Controller/HoldsController.php

```diff
@@ -147,10 +155,26 @@ public function listAction()
 
             $driversNeeded[] = $current;
         }
+        // Cache the current list of requests for editing:
+        $this->putCachedData(
+            $this->getCacheId($patron, 'holds'),
+            $driversNeeded,
+            600
```

Comment:
Is it worth using a class property here and below instead of the magic number "600"?

Reply:
I ended up setting the cacheLifetime property in constructor. Ok or would you prefer a more explicit property?

Reply:
That seems fine for this case; we can sort it out later if we ever need other types of caching in this controller.